src/envoy/http/backend_auth/filter_test.cc

```diff
@@ -86,9 +105,16 @@ TEST_F(BackendAuthFilterTest, NotHaveAudience) {
       filter_->decodeHeaders(headers, false);
 
   ASSERT_EQ(status, Envoy::Http::FilterHeadersStatus::Continue);
+
+  // Stats.
+  const Envoy::Stats::CounterSharedPtr counter =
+      Envoy::TestUtility::findCounter(
+          scope_, "backend_auth.allowed_by_no_configured_rules");
+  ASSERT_NE(counter, nullptr);
+  EXPECT_EQ(counter->value(), 1);
 }
 
-TEST_F(BackendAuthFilterTest, HasAudienceButGetsEmptyToken) {
+TEST_F(BackendAuthFilterTest, HasAudienceButGetsEmptyTokenRejected) {
   Envoy::Http::TestRequestHeaderMapImpl headers{{":method", "GET"},
                                                 {":path", "/books/1"}};
   utils::setStringFilterState(
```

Comment:
Can we reuse these lines of code by putting into function?

Reply:
Done